src/H5Dpublic.h

```diff
@@ -1250,10 +1250,11 @@ H5_DLL herr_t H5Dwrite_chunk(hid_t dset_id, hid_t dxpl_id, uint32_t filters, con
  *                      dataspace
  * \param[in,out]  filters  Mask for identifying the filters in use
  * \param[out]  buf     Buffer containing data to be read from the chunk
+ * \param[in,out]  nalloc   Size of buf in bytes
```

Comment:
Is the naming convention `nalloc`, or `buf_size`? I see both being used. What should be used going forward? For Fortran, I think we mostly use buf_size.

Reply:
I used nalloc because that's what the closest existing analogs for this pattern, H5Tencode etc., use. I do think buf_size is a better name though, so I wouldn't be opposed to changing it.

Reply:
I changed it to `buf_size`.